x-pack/plugins/security_solution/public/common/components/callouts/callout.test.tsx

```diff
@@ -75,7 +75,7 @@ describe('callout', () => {
         <CallOut message={message} onDismiss={onDismiss} />
       </TestProviders>
     );
-    wrapper.find('[data-test-subj="callout-dismiss-btn"]').first().simulate('click');
+    wrapper.find('button[data-test-subj="callout-dismiss-btn"]').first().simulate('click');
```

Comment:
why button is needed here? Shouldn't just data-test-subj attr be enough? I saw button added in similar cases for other tests as well

Reply:
We need to be more specific when targeting buttons because of the way Jest and Enzyme query for elements after components have been converted to Emotion. Without specifying that this element should be a button, the wrapper will not be able to find the element.